tests/tensorflow/nightly/serving.libsonnet

```diff
@@ -19,21 +19,40 @@ local mixins = import 'templates/mixins.libsonnet';
 local tpus = import 'templates/tpus.libsonnet';
 
 {
-  local inference = common.ModelGardenTest {
-    modelName: 'inference',
+  local servingTest = common.ServingTest {
+    local config = self,
     command: [
-      'curl',
-      '-v',
-      '-d',
-      '\'{"instances": [1.0, 2.0, 5.0]}\'',
-      'http://$(cat /scripts/tpu_ip):8501/v1/models/half_plus_two:predict',
+      'python3',
+      'tpu/models/experimental/inference/load_test/examples/loadgen_grpc_main.py',
+      '--target=grpc://$(cat /scripts/tpu_ip):8500',
+      '--model_name=%(model)s' % config,
+      '--scenario=server',
+      '--performance_sample_count=100',
+      '--data_type=%(dataType)s' % config,
+      '--target_latency_percentile=0.99',
+      '--target_latency_ns=100000000',
+      '--batch_size=%(batchSize)d' % config,
+      '--duration_ms=1000',
+      '--query_count=100',
     ],
   },
   local functional = common.Functional,
+  local bert = servingTest {
+    model: 'bert',
+    dataType: 'synthetic_bert',
+    batchSize: 4,
+    gcsDir: 'gs://serving-benchmarks/bert-base-tf2/tpu',
+  },
+  local tpuVm = experimental.TensorflowServingTpuVmMixin,
```

Comment:
Does `tpuVm` need to be structured as a mixin here?  Can ServingTest inherit from `BaseTpuVmTest` and remove the need for TensorFlowServingTpuVmMixin altogether?

Reply:
Merged them together